modules/ROOT/pages/design-publish.adoc

```diff
@@ -1,59 +1,63 @@
+<2> *API version*: The version number of the API specification. For a description of this field, see <<api-version>>.
+<3> *LifeCycle State*: The lifecycle state of the asset. Assets can be published from API Designer as *In Development* or *Stable*. Assets cannot be deprecated from API Designer. 
+<4> *Business group ID*: The ID of the business unit in Anypoint Platform to which the project belongs. 
+<5> *Asset name*: The name that Anypoint Exchange uses to identify the published project. You should typically keep the value as is.
+<6> *Asset ID*: The ID that Anypoint Exchange uses to identify the published project. You should typically keep the value as is.
 +
-<1> *Asset version*: The version number to be displayed in the *Version* section of the page that Anypoint Exchange creates for the project.
-<2> *API version*: The version number of the API specification. For a description of this field, see the "Filling in the *API version* Field" section.
-<3> *Business group ID*: The ID of the business unit in Anypoint Platform to which the project belongs. You should keep the value as it is, except in unusual cases.
-<4> *Asset name*: The name that Anypoint Exchange uses to identify the published project. You should keep the value as is, except in unusual cases.
-<5> *Asset ID*: The ID that Anypoint Exchange uses to identify the published project. You should keep the value as is, except in unusual cases.
-<6> *Add to API Manager*: Specifies that you want to publish the API specification to an API portal. When you click *Publish to Exchange*, API Designer also publishes the specification to API Manager, where the specification is listed in the *API Administration* section.
 . After completing the required fields, click *Publish to Exchange*.
 +
-If your project includes files that are not referenced by the root file either directly or indirectly, a dialog asks you to confirm that you do want to publish the project together with those unreferenced files. By default, API Designer excludes them from the published project in Anypoint Exchange.
+If your project includes files that are not referenced by the root file either directly or indirectly, a dialog asks you to confirm that you  want to publish the project with the unreferenced files. By default, API Designer excludes them from the published project in Anypoint Exchange.
+
+NOTE: If you are using the non-crowd version of Anypoint Platform and want to add the asset to API Manager while publishing to Exchange, select *Add to API Manager*. This functionality is not available in the crowd version and instead you can link API instances in API Manager to assets in Exchange.
```

Comment:
If this refers to Atlassian Crown, then "non-crowd" > "non-Crowd".
Or, "If you are not using the Atlassian Crowd version...."

Reply:
Assuming the above,
```suggestion
NOTE: If you are not using the Atlassian Crowd version of Anypoint Platform and want to add an asset to API Manager while publishing to Exchange, select *Add to API Manager*. If you are using the Crowd version, link API instances in API Manager to assets in Exchange.
```

Reply:
I'm tracking down this question. I searched the doc portal and can't find an explanation of what we mean by crowd other than a comparison of what non-crowd did vs. what crowd does. I'll update this as soon as I have a definitive answer.